src/components/SignUp/SignUp.js

```diff
@@ -18,6 +18,29 @@ export class SignUp extends Component {
       skillSet: ''
     }
   }
+  localSignup = (e) => {
```

Comment:
I dn't think we need this since sign up is already working. Also, the ajax calls are already done in Actions.